packages/icons/utils/internal.js

```diff
@@ -60,6 +60,15 @@ export function __createIcon(name, variants, intrinsicSize = [24, 24]) {
             // To align icon center.
             fontSize: 0,
         }
+        // background image can't be correctly resolved on Firefly in ShadowDOM.
+        // This is a workaround.
+        // https://mask.atlassian.net/browse/FW-564
+        if (iconStyle.backgroundImage && navigator.userAgent.includes('Firefox')) {
```

Comment:
CC @Jack-Works

Reply:
why this can fix the problem? looks like it just repeat the backgroundImage property?

Reply:
Not the same.

With `sx` prop, `mui` creates stylesheet and injects the style in it. In this way, Firefox can not correctly resolve the image. It might be a bug of Firefox that the Stylesheet in the ShadowDOM doesn't inherit the `baseURI` of the document.

<img width="549" alt="image" src="https://github.com/DimensionDev/Maskbook/assets/158803171/dbaa0f8f-9a4e-4af7-a957-6bb90fca3c5d">

Reply:
can you write this in the comment? it will be easier to tell future people why you'r doing this

Reply:
> can you write this in the comment? it will be easier to tell future people why you'r doing this

It's already written above.

Reply:
I mean in the code comment